### PR TITLE
Step description update

### PR DESCRIPTION
**Pull request by mateherber**
### Checklist

- [x] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [x] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires a *MAJOR/MINOR/PATCH* [version update](https://semver.org/)

### Context

New Step description :)

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ workflows:
             set -x
 
             go get -u -v github.com/kisielk/errcheck
-            go get -u -v github.com/golang/lint/golint
+            go get -u -v golang.org/x/lint/golint
```
Comment (lpusok) [inline bitrise.yml]: The check Step could be an easy upgrade, that way lint packages would not be added to go.mod.

Comment (lpusok) [reply bitrise.yml]: Utility workflows + release-config.yml are no longer needed (Depending how it was released last time ofc).

Comment (mateherber) [reply bitrise.yml]: image
:thisisfine:

File: step.yml [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
-  With this step you can copy a value into as many environment variables as you like.
+  Set a value for one or more Environment Variables.
+description: |-
+  With this Step, you can set the same value for as many Environment Variables (Env Vars) as you like. Subsequent Steps in your Workflow can use these Env Vars.
+
+  You can create new Env Vars with this Step and you can also update existing Env Vars with a new value.
+
+  ### Configuring the Step
+
+  1. In the **Value for copy into Environment Variables** input, set the value that you want for your Env Vars.
```
Comment (mateherber) [inline step.yml:12]: This section seems to be a duplicate information. Aren't the input/output descriptions meant to give this guidance? I'm a bit afraid of duplicating input / output guidance as we'll need to keep track of both places when the step changes.

Comment (mateherber) [reply step.yml:12]: @zoltan-baba ☝🏻 ?

**Review by ofalvai — APPROVED** [review]

**Review by lpusok — APPROVED** [review]

**Pull request merged**